app/controllers/fhir/r4/api_controller.rb

```diff
@@ -504,6 +503,7 @@ def check_client_type
         head :unauthorized
       end
     elsif current_client_application.present?
+      Rails.logger.info("Client:Applicaiton, ID:#{current_client_application.id}, Name:#{current_client_application.name}")
```

Comment:
Maybe leave a space between the colon and the id and name - I think that would make it a little easier to scan in the logs

Reply:
Updated here https://github.com/SaraAlert/SaraAlert/pull/707/commits/46afc9977cffe736d394ed995eb0a84b357cd3cd